OWML.Launcher/App.cs

```diff
@@ -59,6 +63,20 @@ public void Run(string[] args)
             Console.ReadLine();
         }
 
+        private void CheckVersion()
+        {
+            var owVersion = _gameVersion.GetVersion();
+            var supportedOwVersion = _owmlManifest.OWVersion;
+            if (owVersion.StartsWith(supportedOwVersion))
```

Comment:
hm. if there was a change that broke OWML from eg. 1.0.7 -> 1.0.7.2 (and the manifest says 1.0.7) this wouldn't hold up...

Reply:
dunno what to do about it tho 🤷